app/src/main/java/org/thoughtcrime/securesms/notifications/MultipleRecipientNotificationBuilder.java

```diff
@@ -40,9 +40,15 @@ public MultipleRecipientNotificationBuilder(Context context, NotificationPrivacy
   }
 
   public void setMessageCount(int messageCount, int threadCount) {
-    setSubText(context.getString(R.string.MessageNotifier_d_new_messages_in_d_conversations,
-                                 messageCount, threadCount));
-    setContentInfo(String.valueOf(messageCount));
+    String txt = Phrase.from(context, R.string.notificationsSystem)
+            .put(MESSAGE_COUNT_KEY, messageCount)
+            .put(CONVERSATION_COUNT_KEY, threadCount)
+            .format().toString();
+    setSubText(txt);
+
+    // Note: `setContentInfo` details are only visible in Android API 24 and below - as our minimum is now API 26 this can be skipped.
+    //setContentInfo(String.valueOf(messageCount));
```

Comment:
If this doesn't relate to anything we are catering for then it can be removed completely instead of commented out.
Same with the rest of content info if it isn't used above api 24

Reply:
Removed